lib/process_settings/testing/helpers.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'process_settings/monitor'
+require 'process_settings/settings'
+require 'process_settings/target_and_settings'
+
+require 'process_settings/testing/monitor'
+
+module ProcessSettings
+  module Testing
+    module Helpers
+      extend self
+
+      TRUE_TARGET = ProcessSettings::Target.new({})
```

Comment:
I was planning to refactor this into the `Target` class, at the bottom. This `{}` constant is used in a few places that could DRY up.

Reply:
I'd like to defer to you on how you'd like to do this.

Reply:
Ok, I can make a PR off this PR.